package.json

```
@@ -8,7 +8,7 @@
     "url": "https://github.com/webpack/webpack-cli.git"
   },
   "engines": {
-    "node": ">=10.13.0"
+    "node": ">=12.13.0"
```

This is the min for ESM support. Same needed for other subpackages
```suggestion
    "node": ">=12.22.0"
```